app/routes/meetings/components/MeetingEditor.tsx

```diff
@@ -252,7 +271,16 @@ const MeetingEditor = () => {
                 name="report"
                 label="Referat"
                 component={EditorField.Field}
+                key={formKey}
               />
+
+              <FormSpy
+                subscription={{ values: true }}
+                onChange={({ values }) => {
+                  setReportValue(values.report || '');
+                }}
+              />
```

Comment:
Now it updates the report whenever anything in the form changes, no? Is it not better to only listen to changes to the report? I sitll think it should be debounced

Reply:
I added debouncing with 4500ms delay now.